### DISCUSSION TITLE
Check use of wet point indexing vs ocean/land mask in iHAMOCC

### DISCUSSION THREAD

**Discussion by TomasTorsvik** [NorESMhub/BLOM#200] [General]
This came up as a question when applying fields defined only for wet points in BLOM inside iHAMOCC routines (PR #179).

In the BLOM part, the wet points are referenced by index counters `isp`, `ifp` and `ilp`. These are used also in iHAMOCC modules `mo_bgcmean` and `mo_intfcblom`. Else where iHAMOCC is using a masking field `omask`.

Do we have a consistent way of handling "wet point" fields in iHAMOCC that have been imported from BLOM? Should we apply `omask` for all these fields, or is there a good reason to keep the mix of wet points and masking as we do today?

Implementation for wet points indexing:
```
   do j=1,jj
      do l=1,isp(j)
         do i=max(1,ifp(j,l)),min(ii,ilp(j,l))
            ...
         enddo
      enddo
   enddo
```

**Comment by JorgSchwinger** [comment]
Inside "core" HAMOCC (i.e. in `hamocc4bcm` and below) we have only used `omask` to indicate wet points. I see no reason to change this.
